arkouda/sorting.py

```diff
@@ -172,5 +175,7 @@ def sort(pda : pdarray) -> pdarray:
     """
     if pda.size == 0:
         return zeros(0, dtype=int64)
+    if pda.dtype not in numeric_dtypes:
+        raise ValueError("ak.sort supports float64 or int64, not {}".format(pda.dtype))
     repMsg = generic_msg("sort {}".format(pda.name))
     return create_pdarray(cast(str,repMsg))
```

Comment:
cast again?, is it needed?

Reply:
yes

Reply:
@mhmerrill good catch! this cast is no longer needed, so I removed 'em